pages/linux/steghide.md

```diff
@@ -0,0 +1,28 @@
+# steghide
+
+> Steganography tool for JPEG, BMP, WAV and AU file formats.
+> More information: <https://github.com/StefanoDeVuono/steghide>.
+
+- Embed data in a PNG image:
+
+`steghide embed --coverfile {{path/to/image.png}} --embedfile {{path/to/data.txt}} --passphrase "{{passphrase}}"`
+
+- Extract data from a WAV audio file:
+
+`steghide extract --stegofile {{path/to/sound.wav}} --passphrase "{{passphrase}}"`
+
+- Display file information, trying to detect an embedded file:
+
+`steghide info {{path/to/file.jpg}}`
+
+- Embed data in a JPG image, compressing it at its maximum:
```

Comment:
Perhaps this will read better?
```suggestion
- Embed data in a JPEG image, using maximum compression:
```

The `E` in `JPEG` is optional, but would make it more consistent with other pages (e.g. `jpegoptim`).